.gitignore

```diff
@@ -7,7 +7,11 @@
 */pnpm-lock.yaml
 */yarn.lock
 
+# Ignore vscode config folder, except in specific templates
+.vscode
+!/gsap-svelte/.vscode
+!/quasar/.vscode
```

Comment:
Erring on the side of ignoring all `.vscode` folders so we don't accidentally commit them, but listing exceptions for templates where we want to commit them.